tests/plugins/test_download_remote_source.py

```diff
@@ -127,6 +128,11 @@ def test_download_remote_source(
                 plugin.run()
             return
 
+        if source_url:
+            (flexmock(DownloadRemoteSourcePlugin)
+                .should_receive('generate_cachito_env_file')
```

Comment:
Can we have unittest that tests content of the created file?

Reply:
preferably with a special value to test check escaping

Reply:
Unit test added, please take a look